README.md

```diff
@@ -1 +1,4 @@
-# Hello-World
\ No newline at end of file
+# Hello-World
+#Project built with HTML and CSS
+
+#It is included linters and gitignore
```

Comment:
- [ ] Please make your README file more descriptive by using [this template](https://github.com/microverseinc/readme-template) and at least edit these sections (Project name, Project description, Getting Started, Built with, Author, and License) with the proper info and please delete any other unused sections.